README.md

```diff
@@ -13,7 +13,7 @@ Interrupts and counters are not currently supported on the board
 
 #### example enabling a PWM pin
 
-If you want to enable pins O_3 and O_9 as a PWM pins, take the following steps
+If you want to enable pins O_3 and O_9 as a PWM pin, take the following steps
```

Comment:
I would have removed the "a" instead of making "pins" singular. You're enabling 2 pins as "a PWM pin" now, which sounds wrong to me. but I'm open to keeping your change as-is if that's what you think it should be.

Reply:
a PWM pin def sounds wrong, and I have it the other way later in the readme. ty for calling out